siloLapisTests/test/aggregatedQueries/regionIsEurope.json

```diff
@@ -0,0 +1,9 @@
+{
+  "testCaseName": "country equals Switzerland",
```

Comment:
The name is wrong? Shouldn't it be "region equals Europe"?

Reply:
Thanks for catching this, I fixed it.